lightning/src/routing/scoring.rs

```diff
@@ -774,6 +793,37 @@ impl<G: Deref<Target = NetworkGraph<L>>, L: Deref, T: Time> ProbabilisticScorerU
 		None
 	}
 
+	/// Query the historical estimated minimum and maximum liquidity available for sending a
+	/// payment over the channel with `scid` towards the given `target` node.
+	///
+	/// Returns the relative probabilities of the liquidity bounds being in each octile,
```

Comment:
I think this needs more explanation, and preferably also a really trivially simple example that breaks down how exactly to interpret the returned `([u16; 8], [u16; 8])`

Reply:
Thank you, they are much better now. However, still see a lot potential for questions arising: if it's a `u16`, why does 32 correspond to 100%? Is it just a coincidence that the given example of 32 is a power of two? If these are probabilities (vs. frequencies), how come they are integer numbers? How would I calculate the actual probabilities? What does being "in a given octile" refer to? Octile of the channel capacity I assume? Do I need to understand what 'buckets' are? etc. pp.

Generally I think this currently still leaks too much of our internal logic/lingo. Can't we here just make it easier for the user and return sets of probabilities, i.e., floats in $[0, 1]$?

Reply:
> I rephrased it again to describe it as frequencies instead of probabilities, and mentioned explicitly that its an arbitrary scale. Users shouldn't attempt to derive meaning in the 32s, as we could change those in the future. I also added a note on how we calculate probabilities.

Thanks, reads much clearer now!

> I would prefer to not massage the data too much, as this is mostly useful for helping users debug in the field, allowing us/users to fetch+log the "raw" data and then use that to figure out why we scored a channel some way after we did it.

Alright, even though I still think floating point numbers would be easier to interpret, the current version should do the job given the verbosity of the docs.